### PR TITLE
Feature/course 2 task 2

### PR DESCRIPTION
**Pull request by grojas123**
All the tasks Done even the optionals .

### CODE REVIEW

File: staff/gustavo-rojas/task2/tgif-project/scripts/html.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+I believe this is by design -- we don't currently allow front-end javascript calls to the API, only backend calls from the server. 
+
+-- 
+Ken Schwencke
+Editor, News Apps
+ProPublica
+For recomendation of mentor I used https://designer.mocky.io/
+
+*/
```
Comment (elolivier) [inline staff/gustavo-rojas/task2/tgif-project/scripts/html.js]: This comment is no needed, please delete it.

File: staff/gustavo-rojas/task2/tgif-project/scripts/html.js [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+function removeallrowstable(table_id) {
+    var table = document.getElementById(table_id);
+
+    for(var i = table.rows.length - 1; i > 0; i--)
+    {
+        table.deleteRow(i);
+    }
+
+}
+//* This is for if we need to wait the to the date to be storage on jsonsenators  .
```
Comment (elolivier) [inline staff/gustavo-rojas/task2/tgif-project/scripts/html.js]: Please rephrase this comment, I'm not sure what is the message.

**Review by elolivier — CHANGES_REQUESTED** [review]

File: staff/gustavo-rojas/task2/tgif-project/scripts/html.js (hunk 1 above)
Comment (grojas123) [reply staff/gustavo-rojas/task2/tgif-project/scripts/html.js]: Removed

File: staff/gustavo-rojas/task2/tgif-project/scripts/html.js (hunk 2 above)
Comment (grojas123) [reply staff/gustavo-rojas/task2/tgif-project/scripts/html.js]: rephrase done .

**Comment by grojas123** [comment]
Done.

**Review by elolivier — APPROVED** [review]

**Pull request merged**